src/firefly/java/edu/caltech/ipac/firefly/messaging/Messenger.java

```diff
@@ -93,6 +97,14 @@ public static String getStats() {
         }
     }
 
+    public static String getRedisHostPortDesc() {
+        String detail= connectAttempt==ConnectAttempt.CONNECTED ? "" :
+                connectAttempt==ConnectAttempt.FAIL_TO_CONNECT ? " (failed to connect)" : " (not using)";
+        return REDIS_HOST + ":" + REDIS_PORT + detail;
```

Comment:
Since the Status enum already contains the description, a toString() of it should be good enough.  init() is needed to ensure it's initialized.
```
init();
return String.format("%s:%s (%s)", REDIS_HOST, REDIS_PORT, status);
```